modulesets/gtk-osx-random.modules

```diff
@@ -35,7 +35,7 @@
   </cmake>
 
   <autotools id="vala" autogen-sh="configure">
-    <branch repo="ftp.gnome.org" version="0.52.0"
+    <branch repo="download.gnome.org" version="0.52.0"
```

Comment:
You made `download.gnome.org` the default repo so you can remove the repo attribute here like you did in modulesets-stable.

Reply:
Oops. That was a mistake!
`modulesets-stable` can switch the default safely but `modulesets` cannot: it uses `git.gnome.org` as default, not 'download.gnome.org'.